src/main/java/com/couchbase/lite/replicator/ReplicationInternal.java

```diff
@@ -857,7 +857,7 @@ public void onCompletion(HttpResponse httpResponse, Object result, Throwable e)
                     // TODO: with regards to behavior of a continuous replication.
                     // Note: was added in order that unit test testRunReplicationWithError() finished and passed.
                     // (before adding this, the replication would just end up in limbo and never finish)
-                    fireTrigger(ReplicationTrigger.STOP_GRACEFUL);  // TODO: call triggerStop(); just to be more consistent
+                    fireTrigger(ReplicationTrigger.STOP_GRACEFUL);  // TODO: call triggerStopGraceful(); just to be more consistent
```

Comment:
I'm confusing between  fireTrigger(ReplicationTrigger.STOP_GRACEFUL) and triggerStopGraceful(). Does it do the same thing? Why can't we call triggerStopGraceful()?